estate/models/estate_property.py

```diff
@@ -0,0 +1,90 @@
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+from odoo import api, fields, models
+
+class RealEstate(models.Model):
+    _name = "estate.property"
+    _description = "RealEstate Model"
+
+    name = fields.Char(required=True,string="Name", default="name")
+    property_type_id = fields.Many2one(comodel_name="estate.property.type", string="Propery Type")
+    buyer_id = fields.Many2one(comodel_name="res.partner", string="Buyer", copy=False)
+    salesperson_id = fields.Many2one(comodel_name="res.users", string="Seles Person",
+        default=lambda self: self.env.uid)
+    tag_ids = fields.Many2many(comodel_name="estate.property.tag", string="Tags")
+    offer_ids = fields.One2many("estate.property.offer", "property_id", string=" ")
```

Comment:
```suggestion
    offer_ids = fields.One2many('estate.property.offer', "property_id", string=" ")
```
and why did you add blank string?